packages/cli/config-management/dependencies-configuration/src/convertDependenciesConfiguration.ts

```diff
@@ -18,20 +22,35 @@ export async function convertDependenciesConfiguration({
     }
 
     const dependencies: Record<string, Dependency> = {};
-    for (const [coordinate, version] of Object.entries(rawDependenciesConfiguration.dependencies)) {
-        const unprefixedCoordinate = coordinate.substring(1);
-        const splitCoordinate = unprefixedCoordinate.split("/");
-        const organization = splitCoordinate[0];
-        const apiName = splitCoordinate[1];
-        if (organization != null && apiName != null) {
-            dependencies[coordinate] = {
-                organization,
-                apiName,
-                version,
-            };
+    for (const [coordinate, versionOrPath] of Object.entries(rawDependenciesConfiguration.dependencies)) {
+        if (isCoordinate(coordinate)) {
```

Comment:
shouldn't we be checking if `versionOrPath` is a version or path? The coordinate is up to them so they might name it `@org/dep` to match other dependencies

Reply:
good point, its what i had before not sure why i changed it

Reply:
i went this route because i figured version could also be set by them